.azure-pipelines/azure-pipelines.yml

```diff
@@ -287,6 +287,8 @@ stages:
         parameters:
           testFormat: 2.16/linux/{0}
           targets:
+            - name: Fedora 38
```

Comment:
Shouldn't this be `Fedora 39`?

Reply:
No, since this is `fedora38` for stable-2.16.